runtime/doc/options.txt

```diff
@@ -1687,6 +1687,16 @@ A jump table for the options with a short description can be found at |Q_op|.
 		:set columns=9999
 <	Minimum value is 12, maximum value is 10000.
 
+						*'columnspace'* *'csp'*
+'columnspace' 'csp'	number	(default 0)
+			global
+			{not in Vi}
+			{only in the GUI}
```

Comment:
This should say "only in the MacVim GUI", unless you can convince upstream to adopt the new option.